examples/ensemble/plot_stacked_generalization_regression.py

```diff
@@ -0,0 +1,81 @@
+"""
+======================================================
+Stacked generalization applied to a regression problem
+======================================================
+
+This example shows how stacked generalization can be used to combine several
+regressors into a single stacked one that performs better than the best
+regressor.
+
+We use Boston's house pricing dataset to compare the mean squared error between
+three regressors (SVM, Lasso and Ridge regressions) and the combination of
+their outputs with a single linear regression. The following result is
+achieved.
+
+MSE for lasso: 22.91279
```

Comment:
This is rendered with reST all on one line. I don't think you really need to include it. It's printed in the rendered example: https://15043-843222-gh.circle-artifacts.com/0/home/ubuntu/scikit-learn/doc/_build/html/stable/auto_examples/ensemble/plot_stacked_generalization_regression.html